src/components/workflow/ViewGraphRequest.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <div id="graph-request-content">
+    <h2>{{ props.id }}</h2>
+    <Card class="graph-request-card">
+      <template #content>
+        <div v-if="loading"><ProgressSpinner /></div>
+        <div v-else class="graph-request">
+          <TaskViewer :id="id" :editMode="editMode" :submitRequested="submitRequested" @updateTask="updateTask" />
+          <div class="field">
+            <label for="graph">Graph being requested</label>
+            <Select
+              v-model="selectedGraph"
+              :options="graphOptions"
+              :class="{ 'p-invalid': graphErrorMessage }"
+              :disabled="!editMode"
+              @blur="showErrorMessages.graph = true"
+            />
+            <small v-if="showErrorMessages.graph && graphErrorMessage" class="p-error">{{ graphErrorMessage }}</small>
+          </div>
+          <div class="flex gap-1">
+            <Button v-if="canEdit && !editMode" label="Edit" @click="editMode = true" />
+            <Button v-if="editMode" label="Cancel" @click="cancelEdit" severity="secondary" />
+            <Button v-if="editMode" @click="updateGraphRequest" :loading="loading" label="Update" />
+            <Button v-if="isAdmin || isAssignee" label="Reject" severity="danger" @click="rejectGraphRequest" />
```

Comment:
should that be an &&? (same for approve)

Reply:
It's so either an admin or the task assignee can reject/approve the graph request.